engine/classes/Elgg/Cli/DatabaseSeedCommand.php

```diff
@@ -63,12 +63,14 @@ protected function command() {
 		_elgg_services()->set('mailer', new \Laminas\Mail\Transport\InMemory());
 
 		$options = [
-			'limit' => (int) $this->option('limit') ?: 20,
+			'limit' => $this->option('limit'),
 			'image_folder' => $this->option('image_folder'),
 			'type' => $this->option('type'),
 			'create_since' => $this->option('create_since'),
 			'create_until' => $this->option('create_until'),
 			'create' => (bool) $this->argument('create'),
+			'interactive' => !(bool) $this->option('no-interaction'),
```

Comment:
you can determine interactive also in the seeder if you have cli_command available

Reply:
I'm already having trouble with passing the CLI command to the seeder service for interaction, so i don't want to move any more of the logic to that service.

Keeping this as is